src/core.jl

```diff
@@ -1,16 +1,18 @@
 export DSSCore
 
 "Module with the low-level API for OpenDSSDirect"
-module DSSCore 
+module DSSCore
 
 if is_windows()
     if Int == Int64
         const dsslib = "$(dirname(@__FILE__))\\..\\deps\\win64\\OpenDSSDirect.DLL"
     else
         const dsslib = "$(dirname(@__FILE__))\\..\\deps\\win32\\OpenDSSDirect.DLL"
     end
-else
-    const dsslib = "$(dirname(@__FILE__))/../deps/libopendssdirect.so"
+elseif is_linux()
```

Comment:
should also check for x86_64 (and ideally that the distro is glibc-based) otherwise this binary isn't going to work